app/src/main/java/com/example/farmeraid/sign_in/views/SignInScreenView.kt

```diff
@@ -0,0 +1,113 @@
+package com.example.farmeraid.sign_in.views
+
+import android.annotation.SuppressLint
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.ClickableText
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material3.Button
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextField
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.text.input.PasswordVisualTransformation
+import androidx.compose.ui.text.input.TextFieldValue
+import androidx.compose.ui.text.style.TextDecoration
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.example.farmeraid.sign_in.SignInViewModel
+//REsource used for design : https://medium.com/@manojbhadane/android-login-screen-using-jetpack-compose-part-2-a262ad87c6d
+@SuppressLint("UnusedMaterial3ScaffoldPaddingParameter")
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun SignInScreenView() {
+    val viewModel = viewModel(SignInViewModel::class.java)
+    val state by viewModel.state.collectAsState()
+//    val viewModel = viewModel(HomeViewModel::class.java)
+//    val state by viewModel.state.collectAsState()
+//
+//    Scaffold(
+//        floatingActionButton = {
+//            OutlinedButtonView(
+//                buttonUiState = state.buttonUiState,
+//                buttonUiEvent = UiComponentModel.ButtonUiEvent(
+//                    onClick = { viewModel.changeSelectedTab(if (state.selectedTab == HomeModel.Tab.Inventory) HomeModel.Tab.Quotas else HomeModel.Tab.Inventory) }
+//                )
+//            )
+//        },
+//    ) {
+//        Text(state.selectedTab.toString())
+//    }
+    Box(modifier = Modifier.fillMaxSize()) {
+        ClickableText(
+            text = AnnotatedString("Sign up here"),
+            modifier = Modifier
+                .align(Alignment.BottomCenter)
+                .padding(20.dp),
+            onClick = { },
```

Comment:
Add a TODO comment here so we know that we need to add the onClick for this later